string name
---
float64 x_axis_center
float64 width
float64 height